src/qibo_cloud_backends/braket_client.py

```diff
@@ -0,0 +1,79 @@
+import time
+
+from braket.aws import AwsDevice
+from braket.devices import LocalSimulator
+from qibo import Circuit as QiboCircuit
+from qibo.backends import NumpyBackend
+from qibo.config import raise_error
+from qibo.result import MeasurementOutcomes
+
+from qibo_cloud_backends.braket_translation import to_braket
+
+
+class BraketClientBackend(NumpyBackend):
+    def __init__(self, device=None, verbatim_circuit=False, verbosity=False):
```

Comment:
I'm not sure if it's a good idea to leave `token` argument in just for symmetry with other cloud backends, because Braket doesn't need the `token` argument. A user may be confused and wonder if there will be a feature for `token` in future. What do you think? :)

Reply:
I know that it may create some confusion, but I ultimately think that it would be better to uniform all the backends under the input arguments perspective. It should be fine, as long as this is well documented and/or warnings are raised for unused arguments.

Reply:
Once more: refrain from warnings if you're not directly writing an app (i.e. a CLI or a graphical app).

Choose your favorite among:
- ignoring
- raise an error
- return some marker (possibly together with other values)

Reply:
I might choose ignore because `raise_error` will stop the code from execution. But let me try all the options out on my end. Thanks a lot!